### DISCUSSION TITLE
Copilot for Students stops working again after renewal

### DISCUSSION THREAD

**Discussion by Aditya-Binukumar** [community/community#44067] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

I am a verified student with access to the GitHub Student Development Pack and recently got my Copilot access renewed after its access expired on **9th Jan 2023**. #43634 

My Copilot access is currently set to expire next on **8th, Feb 2023**.

![image](https://user-images.githubusercontent.com/91012858/211973653-80235b18-0b27-4572-a55f-a1a79e96d432.png)

Despite having renewed access, I am getting the same error that I got when it expired on the 9th ("No access to GitHub Copilot found."). This error showed up only a few minutes ago.

![image](https://user-images.githubusercontent.com/91012858/211972383-04fc68d6-939e-437e-a1fa-257330852b15.png)

The error seems to persist despite logging out and logging back in, for both the IDE and the extensions. 

Please let me know what I can do to fix this issue.

**Comment by Linloir** [comment]
same issue. lost access after renew

**Comment by Aditya-Binukumar** [comment, marked as answer]
It seems like my Copilot access has returned now, not sure how it went, though thank you Copilot team for fixing this bug.

**Comment by VineetTambe** [comment]
I am facing the same issue - After I click submit on the reverification page, the page freezes and nothing happens.

**Comment by JuGit-pk** [comment]
i am still following that issue, is there is any update ?

**Reply by tuancuong16** [reply]
I am facing this problem too. My plan is still Github Pro but the Copilot is not enabled and force me have to subscribe
